### ISSUE TITLE
Arc mark (for donuts and pies).

### ISSUE THREAD

**Issue by Fil** [observablehq/plot#80]
Here's a nice(?) example found in the wild, of faceted donuts.

![image](https://github.com/user-attachments/assets/8fc79760-a374-4e72-b522-e425d458586c)

(originally from https://immunology.sciencemag.org/content/immunology/5/54/eabf3698/F1.large.jpg)

[_Source_](https://immunology.sciencemag.org/content/5/54/eabf3698/tab-figures-data)

The data is available, but I don't think it's the right dataset to use, since I don't understand what it means :)

**Comment by Fil** [comment]
see also https://github.com/observablehq/plot/issues/133

**Comment by spiderdab** [comment]
I think adding an Arc mark should be a priority.

**Comment by mbostock** [comment]
@spiderdab Please react to the top post with a 👍 to upvote an issue. Thank you.

**Comment by matys1** [comment]
It would be nice for this to finally land.

**Comment by codewizard-dt** [comment]
Yeah why is this still on hold for 4 years?  Do we need contributors?  I'd be happy to help

**Comment by NicoleRauch** [comment]
Is there anything planned for this? 🤔
